### DISCUSSION TITLE
Support filtering out deprecated packages from organisation/user package lists

### DISCUSSION THREAD

**Discussion by ThisIsMissEm** [npm/feedback#660] [Website] [answered]
Currently I've an organisation that has numerous deprecated packages, and it'd be good to have those hidden by default, as they're not meant to be being used.

**Comment by monishcm** [comment]
Can you elaborate a bit? There is already a checkbox to do it.

![image](https://user-images.githubusercontent.com/23190045/161195560-e0187e9c-7cca-419e-b2b2-7d9aaf38847c.png)

**Reply by ljharb** [reply]
That's for deprecated *versions* - i think the ask is for *packages*. eg, on a page like https://npmjs.com/~ljharb, or in search results.

**Reply by ThisIsMissEm** [reply]
Yes, exactly @ljharb — I was looking in https://www.npmjs.com/settings/inrupt/packages (for my organisation) and in https://www.npmjs.com/org/inrupt

**Reply by monishcm** [reply]
Thanks for the feedback. We've put this in our backlog and will get back to you if we require more information.

**Comment by monishcm** [comment, marked as answer]
Thanks for the feedback. We've put this in our backlog and will get back to you if we require more information.
